src/app/components/source-graphs/source-graphs.component.ts

```diff
@@ -35,6 +35,10 @@ import { Source } from '../../shared/store/source/source.model'
         </div>
       </div>
     </div>
+    <div *ngIf="sources && sources[0].sensors[0]" class="axis">
+      <app-graph-external-x-axis [sensor]="sources[0].sensors[0]"
```

Comment:
For the data for the x axis, I just got the data of the first sensor of the first source? Is that alright?